examples/mobile/HomeApp/js/app.js

```diff
@@ -125,6 +126,24 @@ import Storage from "./clipping-storage.js";
 				(app2.isActive) ? 0 : -1 : 1
 		});
 
+		updateManifests();
+
+		appsList = appsList.forEach((app) => {
+			if (app.isActive) {
+				const appWebClips = new Map();
+
+				app.webClipsList.forEach((webclip) => {
+					appWebClips.set(webclip.url, manifestsList[webclip.url]);
+				});
+
+				Promise.all(appWebClips).then((value) => {
+					app.webClipsList.sort((a, b) => {
+						return (value[a.url] > value[b.url]) ? b : a;
```

Comment:
I don't understand reason of  sort by `url` property. I thought we want sort by card type "service" | "control".